### DISCUSSION TITLE
请问2.6到2.6.1升级，需要更换css和js吗？

### DISCUSSION THREAD

**Discussion by dianso** [ArtalkJS/Artalk#576] [General]
请问2.6到2.6.1升级，需要更换css和js吗？

我是用的二进制部署的，css里面修改了不少颜色，每次升级大版本我都需要对比新旧CSS重新修改。

请问小版本更新修改了css吗？

**Comment by qwqcode** [comment]
2.6.0 到 2.6.1 的升级没有修改前端 CSS 和 JS，感谢支持
